1. CWE Base Mod/Cold War Era Mod (CWE)/common/goods/00_goods.txt

```diff
@@ -777,7 +775,7 @@ circuit_boards = { #Needed for electronic products
 
 semiconductors = { #Needed for electronic products
 	texture = "gfx/interface/icons/goods_icons/semiconductors.dds"
-	cost = 20
+	cost = 30
```

Comment:
Base prices should be kept uniform at 10 for raw resource, 20 for intermediate and 40 for completed products. This should not be an issue as actual prices will be determined by market forces anyway.